cesiumpy/extension/geocode.py

```diff
@@ -9,7 +9,7 @@
 import cesiumpy.util.common as com
 
 # ToDo: want different geocoders?
-_GEOCODER = GoogleV3()
+_GEOCODER = GeocodeFarm()
```

Comment:
This gives me a

`NameError: name 'GeocodeFarm' is not defined`

Add the import to the beginning